corehq/ex-submodules/casexml/apps/phone/models.py

```diff
@@ -609,31 +613,24 @@ def __repr__(self):
         return json.dumps(self.indices, indent=2)
 
     @staticmethod
-    def get_all_dependencies(case_id, child_index_tree, extension_index_tree, closed_cases=None,
+    def get_all_dependencies(case_id, child_index_tree, extension_index_tree,
                              cached_child_map=None, cached_extension_map=None):
-        """Takes a child and extension index tree and returns returns a set of all dependencies of <case_id>
+        """
+        Takes a child and extension index tree and returns returns a set of all dependencies of <case_id>
 
-        Traverse each incoming index, return each touched case. Stop traversing
-        incoming extensions if they lead to closed cases.
```

Comment:
I'm not sure why this condition was here in the first place. @proteusvacuum do you know?